README.md

```diff
@@ -53,6 +53,8 @@ ember serve
 
 Visit [http://localhost:4200](http://localhost:4200)
 
+Note: If you get an error on Mac OSX mentioning `"Error: EMFILE: too many open files, watch"`, try installing Watchman. Install [Homebrew](https://brew.sh/) if you don't have it. Then in you terminal, run `brew update`, then `brew install watchman`.
```

Comment:
```suggestion
Note: If you get an error on Mac OSX mentioning `"Error: EMFILE: too many open files, watch"`, try installing Watchman. Install [Homebrew](https://brew.sh/) if you don't have it. Then in you terminal `brew install watchman`.
```

brew nowadays does an update before trying to install, so it shouldn't be necessary!